src/components/EditTask/EditTask.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux'
+
+import { editTask, updateTask } from '../../actions/'
+import Input from '../Input/Input'
+import Textarea from '../Textarea/Textarea'
+
+import './EditTask.css'
+
+const EditTask = ({ id, title, description, editTask, updateTask }) => {
+  let newTitle = ''
+  let newDescription = ''
+
+  function handleSubmit(event) {
+    event.preventDefault()
+    if (!newTitle.value.trim()) {
+      return
+    }
+
+    updateTask({
+      id,
+      title: newTitle.value,
+      description: newDescription.value,
+    })
+
+    newTitle.value = ''
```

Comment:
`newTitle` should be a string, not an object, right?

Reply:
It's referring to a node not the value of a node, so it is an object. In a better version the input's state would be handled by the store, but unfortunately I've ran out of time, so I did a poor man's solution.